src/components/Settings/SettingsContent.jsx

```diff
@@ -1,206 +1,98 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import flag from 'cozy-flags'
 
-import { isMobileApp } from 'cozy-device-helper'
-import { useClient } from 'cozy-client'
+import { useInstanceInfo } from 'cozy-client'
+import {
+  makeActions,
+  divider
+} from 'cozy-ui/transpiled/react/ActionsMenu/Actions'
+import ActionsMenu from 'cozy-ui/transpiled/react/ActionsMenu'
+import ActionsItems from 'cozy-ui/transpiled/react/ActionsMenu/ActionsItems'
 
-import Icon from 'cozy-ui/transpiled/react/Icon'
-import OpenwithIcon from 'cozy-ui/transpiled/react/Icons/Openwith'
-import PeopleIcon from 'cozy-ui/transpiled/react/Icons/People'
-import PaletteIcon from 'cozy-ui/transpiled/react/Icons/Palette'
-import GraphCircleIcon from 'cozy-ui/transpiled/react/Icons/GraphCircle'
-import CozyCircleIcon from 'cozy-ui/transpiled/react/Icons/CozyCircle'
-import HandIcon from 'cozy-ui/transpiled/react/Icons/Hand'
-import DevicesIcon from 'cozy-ui/transpiled/react/Icons/Devices'
-import GlobeIcon from 'cozy-ui/transpiled/react/Icons/Globe'
-import LogoutIcon from 'cozy-ui/transpiled/react/Icons/Logout'
-import HelpIcon from 'cozy-ui/transpiled/react/Icons/Help'
-import EmailIcon from 'cozy-ui/transpiled/react/Icons/Email'
-
-import { getSettingsLink } from 'components/Settings/helper'
-import StorageData from 'components/Settings/StorageData'
+import {
+  profile,
+  appearance,
+  plans,
+  storage,
+  permissions,
+  connectedDevices,
+  connections,
+  help,
+  contact,
+  logout
+} from 'components/Settings/actions'
 import useI18n from 'components/useI18n'
 
-const MenuIcon = ({ icon }) => {
-  return <Icon className="u-mr-1" color="var(--iconTextColor)" icon={icon} />
-}
+const SettingsContent = ({
+  anchorRef,
+  instanceInfo,
+  onLogOut,
+  isDrawer,
+  onClose
+}) => {
+  const { t } = useI18n()
+
+  const actions = makeActions(
+    [
+      profile,
+      appearance,
+      plans,
+      storage,
+      permissions,
+      divider,
+      connectedDevices,
+      connections,
+      divider,
+      help,
+      contact,
+      logout
+    ],
+    {
+      t,
+      instanceInfo,
+      onLogOut
+    }
+  )
+
+  if (isDrawer) return <ActionsItems actions={actions} />
 
-const ExternalLinkIcon = () => {
   return (
-    <Icon
-      className="coz-nav-settings-item-btn-external-icon"
-      color="var(--iconTextColor)"
-      icon={OpenwithIcon}
+    <ActionsMenu
+      ref={anchorRef}
+      open={true}
+      docs={[]}
```

Comment:
A point for cozy-ui, it would be cool to make this property optional

Reply:
faut que je regarde / test, c'est peut-être déjà le cas